src/main/msp/msp.c

```diff
@@ -1500,13 +1500,13 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, 0);
 #endif
         sbufWriteU8(dst, rxConfig()->fpvCamAngleDegrees);
-        sbufWriteU8(dst, rxConfig()->rcInterpolationChannels);
+        sbufWriteU8(dst, 0); // not required in API 1.44, was rxConfig()->rcSmoothingChannels
 #if defined(USE_RC_SMOOTHING_FILTER)
-        sbufWriteU8(dst, rxConfig()->rc_smoothing_type);
-        sbufWriteU8(dst, rxConfig()->rc_smoothing_input_cutoff);
-        sbufWriteU8(dst, rxConfig()->rc_smoothing_derivative_cutoff);
-        sbufWriteU8(dst, rxConfig()->rc_smoothing_input_type);
-        sbufWriteU8(dst, rxConfig()->rc_smoothing_derivative_type);
+        sbufWriteU8(dst, rxConfig()->rc_smoothing_mode);
```

Comment:
I am concerned about the re-use of this byte - `rc_smoothing_type` is defined differently from `rc_smoothing_mode`, so in order to not risk getting problematic configurations from legacy applications, we should be discarding this byte, and add one for `rc_smoothing_mode` at the end  of the frame.

Reply:
I've thought about this at length and thing the risk is exceedingly low if we leave it as is.  I say this because:
- use of the old linear interpolation smoothing mode mode is vanishingly rare
- it hasn't been default for some years now, and no-one has every encouraged users to go back or try the old method
- the user name has changed in CLI, so copy and paste updaters who used linear interpolation will get safe filters
- only people who upgraded without erasing flash and were using linear interpolation would have a problem.  Not having RC smoothing isn't fatal; it would be the least of their problems. 
Hence I personally feel we can leave it as is.

Reply:
will fix MSP as advised, thanks.